Jenkinsfile-nightly

```diff
@@ -16,7 +17,13 @@ pipeline {
         stage('Performance') {
             agent { label 'benchmark' }
             steps {
-                loop_terminals_run_oltp(OLTP_TERMINALS)
+                sh 'echo $NODE_NAME'
+                sh 'echo y | sudo ./script/installation/packages.sh all'
+                sh 'mkdir build'
+                sh 'cd build && cmake -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_BUILD_TYPE=Release -DTERRIER_USE_ASAN=OFF -DTERRIER_USE_JEMALLOC=ON -DTERRIER_BUILD_TESTS=OFF .. && make -j$(nproc) all'
+                sh "cd build && python3 ../script/testing/oltpbench/run_oltpbench.py --config-file=../script/testing/oltpbench/configs/nightly.json --build-type=release --publish-results=prod --publish-username=${PSS_CREATOR_USR} --publish-password=${PSS_CREATOR_PSW}"
```

Comment:
I would add timeout here just to make sure that things don't end up getting stuck.

Reply:
Done